cmd/api/main.go

```diff
@@ -136,7 +140,29 @@ func responseJSON(w http.ResponseWriter, status int, body any) {
 	}
 }
 
-/* Verifies with all fields are correctly filled and update the book in the db. */
+/* Set 'true' to column 'archived' in the database */
```

Comment:
This does not look as a HTTP handler comment, looks more like a database method comment.
This handler should get more business oriented comments.